### PR TITLE
Assignment 1

### PR DESCRIPTION
**Pull request by rutaihwa**
Basic server

### CODE REVIEW

**Comment by rutaihwa** [comment]
Basic server, not very fancy.

File: index.js [hunk 1]
```diff
@@ -0,0 +1,53 @@
...
+
+server.connection({ port: process.env.PORT || 8000 });
+
+// Routes
+
+server.route({
+    method: 'GET',
+    path: '/version',
+    config:{
+
```
Comment (idanwe) [inline index.js]: You can remove the empty lines here you are defining object.
And add spacing before `function` arguments and between `: {`

Comment (rutaihwa) [inline index.js]: Thanks, @idanwe I am getting the hang of the style guide.

File: index.js [hunk 2]
```diff
@@ -0,0 +1,48 @@
...
+// Create a server
+var server = new Hapi.Server();
+
+// Creates a connection
+server.connection({ port: process.env.PORT || 8000 });
+
+// Routes
+server.route({
+
```
Comment (hueniverse) [inline index.js]: No need for empty line here.

File: index.js [hunk 3]
```diff
@@ -0,0 +1,48 @@
...
+    method: 'GET',
+    path: '/version',
+    config: {
+
```
Comment (hueniverse) [inline index.js]: Same here. No need for extra empty lines.

**Pull request closed without merging**